problems/2.functions.js

```diff
@@ -20,17 +20,18 @@ function greeting() {
 * @returns {null}
 * @modifies {global.globalCount}
 */
-function incrementGlobalCount() {
 
-}
+function incrementGlobalCount() {}
```

Comment:
To update a variable you need to do reassignment and thus need the assignment operator (=). 
global.globalCount += val